src/fsharp/FSharp.Compiler.Service.ProjectCrackerTool.netcore/FSharp.Compiler.Service.ProjectCrackerTool.netcore.fsproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
```

Comment:
as a note, @dsyme i think we should kill ProjectCrackerTool.netcore. is not used or useful afaik

new fsproj use a differnt way.
old fsproj use full framework